README.md

```diff
@@ -45,15 +45,19 @@ Then use the view in your layouts:
-| touchRadius                | rsb_touchRadius            | dimension | The acceptable touch radius around thumbs in pixels                                    |
-| minThumbDrawable           | rsb_minThumbDrawable       | reference | The drawable to draw min thumb with                                                    |
-| maxThumbDrawable           | rsb_maxThumbDrawable       | reference | The drawable to draw max thumb with                                                    |
-| minRange                   | rsb_minRange               | integer   | The minimum range to be selected. It should at least be 1                              |
-| max                        | rsb_max                    | integer   | The maximum value of thumbs which can also be considered as the maximum possible range |
\ No newline at end of file
+| Variable                   | XML Attribute                | Type      | Description                                                                            |
+| :------------------------- | :--------------------------- | :-------- | :--------------------------------------------------------------------------------------|
+| trackColor                 | rsb_trackColor               | color     | Color of horizontal track                                                              |
+| trackSelectedColor         | rsb_trackSelectedColor       | color     | Color of the selected range of horizontal track                                        |
+| trackThickness             | rsb_trackThickness           | dimension | The thickness of the horizontal track                                                  |
+| trackSelectedThickness     | rsb_trackSelectedThickness   | dimension | The thickness of the selected range of horizontal track                                |
+| sidePadding                | rsb_sidePadding              | dimension | Side padding for view, by default 16dp on the left and right                           |
+| touchRadius                | rsb_touchRadius              | dimension | The acceptable touch radius around thumbs in pixels                                    |
+| minThumbDrawable           | rsb_minThumbDrawable         | reference | The drawable to draw min thumb with                                                    |
+| maxThumbDrawable           | rsb_maxThumbDrawable         | reference | The drawable to draw max thumb with                                                    |
+| minRange                   | rsb_minRange                 | integer   | The minimum range to be selected. It should at least be 1                              |
+| max                        | rsb_max                      | integer   | The maximum value of thumbs which can also be considered as the maximum possible range |
+| minThumbOffset             | rsb_minThumbOffsetVertical   | integer   | Vertical pixel offset of min thumb                                                     |
```

Comment:
I would also add this to the sample app.
That way, we can use the sample app as a sort-of show-all-features project (and also, to test the features).